spring-cloud-stream-binder-kafka-docs/src/main/asciidoc/overview.adoc

```diff
@@ -309,12 +308,11 @@ If the topics required already exist on the broker, or will be created by an adm
 
 [source]
 ----
- java -Djava.security.auth.login.config=/path.to/kafka_client_jaas.conf -jar log.jar \
-   --spring.cloud.stream.kafka.binder.brokers=secure.server:9092 \
+ java --spring.cloud.stream.kafka.binder.brokers=secure.server:9092 \
    --spring.cloud.stream.kafka.binder.zkNodes=secure.zookeeper:2181 \
    --spring.cloud.stream.bindings.input.destination=stream.ticktock \
    --spring.cloud.stream.kafka.binder.autoCreateTopics=false \
```

Comment:
If I have a working topic I was able to run apps without manual jaas file. Trying to hook to a topic which would get created will not work. Eventually tried to add `--spring.cloud.stream.kafka.binder.zkJaas.options.serviceName=zookeeper` but it didn't help anything.

So it's not able to create a topic with a leader as seen from here:
```
[root@sandbox bin]# ./kafka-topics.sh --zookeeper sandbox.hortonworks.com:2181 --describe --topic ticktock5
Topic:ticktock5	PartitionCount:1	ReplicationFactor:1	Configs:
	Topic: ticktock5	Partition: 0	Leader: none	Replicas: 1001	Isr: 
[root@sandbox bin]# ./kafka-topics.sh --zookeeper sandbox.hortonworks.com:2181 --describe --topic ticktock1
Topic:ticktock1	PartitionCount:1	ReplicationFactor:1	Configs:
	Topic: ticktock1	Partition: 0	Leader: 1001	Replicas: 1001	Isr: 1001
```
app just logged:
```
2016-09-20 13:22:47.247  WARN 1005 --- [ad | producer-1] org.apache.kafka.clients.NetworkClient   : Error while fetching metadata with correlation id 0 : {ticktock5=LEADER_NOT_AVAILABLE}
2016-09-20 13:22:47.445  WARN 1005 --- [ad | producer-1] org.apache.kafka.clients.NetworkClient   : Error while fetching metadata with correlation id 1 : {ticktock5=LEADER_NOT_AVAILABLE}
2016-09-20 13:22:47.547  WARN 1005 --- [ad | producer-1] org.apache.kafka.clients.NetworkClient   : Error while fetching metadata with correlation id 2 : {ticktock5=LEADER_NOT_AVAILABLE}
```